File: Cs/SampleApp/Form/Model/InputDialog.cs
using System;

namespace SampleApp.Form.Model
{

    /// <summary>
    /// InputDialog.Model
    /// </summary>
    public class InputDialog : IDisposable
    {

        #region ViewModel.Property

        /// <summary>
        /// テンキー1
        /// </summary>
        public string NumText1 = "";

        /// <summary>
        /// テンキー2
        /// </summary>
        public string NumText2 = "";

        /// <summary>
        /// 文字入力1
        /// </summary>
        public string StringText1 = "";

        /// <summary>
        /// 文字入力2
        /// </summary>
        public string StringText2 = "";

        #endregion

        /// <summary>
        /// InputDialog.Model
        /// </summary>
        public InputDialog()
        { }

        /// <summary>
        /// 終了処理
        /// </summary>
        public void Dispose()
        { }

    }

}

File: VB/ScreenKeyboard/SampleProject/Form/Model/InputDialog.vb
Imports System

Namespace Form.Model

    ''' <summary>
    ''' InputDialog.Model
    ''' </summary>
    Public Class InputDialog
        Implements IDisposable

#Region "ViewModel.Property"

        ''' <summary>
        ''' テンキー１
        ''' </summary>
        Public NumText1 As String = ""

        ''' <summary>
        ''' テンキー２
        ''' </summary>
        Public NumText2 As String = ""

        ''' <summary>
        ''' 文字入力１
        ''' </summary>
        Public StringText1 As String = ""

        ''' <summary>
        ''' 文字入力２
        ''' </summary>
        Public StringText2 As String = ""

#End Region

        ''' <summary>
        ''' InputDialog.Model
        ''' </summary>
        Public Sub New()

        End Sub

        ''' <summary>
        ''' 終了処理
        ''' </summary>
        Public Sub Dispose() Implements IDisposable.Dispose

        End Sub

    End Class

End Namespace

